Podfile.lock

```diff
@@ -1,16 +1,16 @@
 PODS:
-  - inai-pod (0.1.17)
+  - inai-pod (0.1.22)
 
 DEPENDENCIES:
-  - inai-pod (= 0.1.17)
+  - inai-pod (= 0.1.22)
 
 SPEC REPOS:
   trunk:
     - inai-pod
 
 SPEC CHECKSUMS:
-  inai-pod: 75ed6b4671e8d8d07be11dba87ff8c281541c590
+  inai-pod: daa249d80f0cf6d2b6efa76daf4f048d9db8034b
```

Comment:
Do we need these files to be pushed?

Reply:
This file contains latest pod version, as we need new features which needs latest version this filed is pushed

Reply:
ya but this actual version is added in the podfile that is enough right? why this podfile.lock needed?

Reply:
In case user is updating the existing local repo, unless user does pod update or remove .lock file user will encounter error

Reply:
I have removed this file